app/assets/stylesheets/config/jquery-ui-assets.css.scss

```diff
@@ -1,33 +1,34 @@
 .ui-icon {
   width: 16px;
   height: 16px;
-  background-image: image-url("ui-icons_222222_256x240.png");
+  background-image: url("../../images/ui-icons_222222_256x240.png");
```

Comment:
so for styles used by JS the path looks like this but for styles used by Rails it's still image-url? or this works too?

Reply:
This works only with Webpack (https://www.npmjs.com/package/resolve-url-loader). That's why we should prolly move files around and renome

Reply:
ah. so we can't share styles :(

Reply:
Depends on the case ;)